skyve-war/src/main/java/modules/admin/Content/Content.xml

```diff
@@ -3,6 +3,7 @@
 			xmlns="http://www.skyve.org/xml/document" 
 			xsi:schemaLocation="http://www.skyve.org/xml/document ../../../schemas/document.xsd" 
 			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<documentation>Stores file and multimedia content metadata with hierarchical organization by customer, module, document, and attribute, tracking content relationships and modification timestamps.</documentation>
```

Comment:
What is this document used for, I've never noticed it before. It is non-persistent. Where is it used?

Reply:
Maybe just add "Non-persistent document used by datamaintenance to store file and multimedia..."